render/egl.c

```diff
@@ -248,28 +265,28 @@ struct wlr_egl *wlr_egl_create(EGLenum platform, void *remote_display,
 			"eglUnbindWaylandDisplayWL");
 		load_egl_proc(&egl->procs.eglQueryWaylandBufferWL,
 			"eglQueryWaylandBufferWL");
+	} else {
+		if (platform == EGL_PLATFORM_DEVICE_EXT) {
```

Comment:
This can be an `else if` block.

Would be nice to explain why we `return false` here: we expect the caller to fallback to the GBM platform.